content/en/logs/indexes/_index.md

```diff
@@ -20,54 +20,74 @@ further_reading:
 
-Order matters for exclusion filters. Contrary to how several pipelines can process a log, if a log matches several exclusion filters, only the first exclusion filter rule is applied.
+{{< img src="logs/indexes/multi_index.png" alt="Multi indexes"  style="width:70%;">}}
 
-Reorder your pipeline to make sure the proper exclusion filters apply to your log. For instance, you probably want to set up the filters ordered by least inclusive to most inclusive queries..
+Use drag and drop on the list of indexes to reorder them.
+
+
+## Exclusion Filters
 
-To reorder your exclusion filter, drag and drop them into your preferred order.
+By default, logs indexes come without exclusion filter: meaning all logs matching the [Index Filter](#indexes-filters) are eventually indexed.
+
+But because your logs are not all and equally valuable, and because some logs are useful only under some circumstances, exclusion filters control which logs flowing in are eventually indexed. Excluded logs are discarded from indexes, but would still flow through the [Livetail][12], [Metrics Generation][13] and/or [Archives][14].
+
+Add as many exclusion filters as you need to control your flow. Exclusion filters come with a query, a sampling rule and a active/inactive toggle:
+
+* Default **query** is `*`, meaning all logs flowing in the index (that is to say logs that matched the index filter). Scope down exclusion filter to only a subset of the logs if you need finer control.  
+* Default **sampling rule** is `Exclude 100% of logs` matching the query. Adapt sampling rate from 0% to 100%, and decide if the sampling rate applies on individual logs, or group of logs defined by the unique values of any attribute. . 
+* Default **toggle** is active, meaning the logs flowing in are actually discarded according to the rule. Toggle this to inactive to ignore this exclusion filter for new logs flowing in the index. 
+
+As for Index Filters, logs will be processed by the only first **active** exclusion filter they match. Which means that order matters: use drag and drop to reorder exclusion filter accordingly.
 
 {{< img src="logs/indexes/reorder_index_filters.png" alt="reorder index filters"  style="width:80%;">}}
 
-### Enable/Disable filters
 
-If not all logs are worth indexing on a daily basis, they might still be important in certain situations.
-Debug logs, for instance, are not always useful, but during complex troubleshooting or a production release, they can become very helpful.
+### Examples of Exclusion Filters
 
-Instead of changing your application logging level or using a complex internal filtering tool, you can change what is indexed directly with Datadog index filters.
+You don't need your DEBUG logs... until you actually need them when your platform undergoes an incident, or want to carefully observe the deployment of a critical version of your application. Setup a 100% exclusion filter on the `status:DEBUG`, and toggle it on and off from UI or [API] when required.
```

Comment:
The link to the API is missing.

```suggestion
You don't need your DEBUG logs... until you actually need them when your platform undergoes an incident, or want to carefully observe the deployment of a critical version of your application. Setup a 100% exclusion filter on the `status:DEBUG`, and toggle it on and off from Datadog UI or through the [API] when required.
```

Reply:
Do we need a link on [API] ?

Reply:
Also, "you don't need" seems a bit strong, should we rephrase with something like `you might not need` ?